.estafette.yaml

```diff
@@ -266,28 +277,30 @@ stages:
         versionTagSuffix: v6.0-sdk
         repositories:
         - extensions
-  
+
       vulnerability-scan-6.0:
         image: extensions/docker:stable
         action: trivy
+        severity: critical
         repositories:
         - eu.gcr.io/travix-com
         container: dotnet
         versionTagSuffix: v6.0-sdk
         repositories:
         - extensions
-      
+
       check-efficiency-7.0:
         image: extensions/docker:stable
         action: dive
         container: dotnet
         versionTagSuffix: v7.0-sdk
         repositories:
         - extensions
-  
+
       vulnerability-scan-7.0:
         image: extensions/docker:stable
         action: trivy
+        severity: critical
```

Comment:
Do we still need to change the severity to critical when we are already using the trivyignore file?

Reply:
We have a bunch of `HIGH` vulnerabilities too. If you prefer, I can keep the limit `high`, and add all of those to `.trivyignore` instead. But this seemed simpler.

Reply:
LGTM

Reply:
Ah ok. Then its fine. We can merge this